### ISSUE TITLE
Newer qemu-kvm packages for CentOS

### ISSUE THREAD

**Issue by tobzsc** [OpenNebula/one#1604]
# Enhancement Request
## Description
Install centos SIG virt packages on centos for newer qemu-kvm:
Just two small line have to be added to docs:
yum install centos-release-qemu-ev
yum update

This is just for virtualization hosts! It solves several bugs especially with Windows VMs where we had to apply some ugly hacks.

## Use case
CentOS supports backports of the redhat enterprise virtualization packages. These provide a more actual version of qemu-kvm on CentOS servers.

## Interface Changes
Nothing, just documentation.

**Comment by tobzsc** [comment]
Why aren't you adding a direct version for such an improvement? I think we should see something like this in the coming version 5.6. Even in a minor version such an improvement would be great.

It is just a few lines of documentation but a huge improvement for the whole community.

**Comment by Madko** [comment]
You can also use centos-release-openstack-pike (or any openstack version) that provides openvswitch package. This centos-release-openstack-pike also install the qemu-ev repository. With this 2 repos, you get the same stack (rpm versions) used in RDO so you get a huge community of testers, users, bug fixes etc

**Comment by tobzsc** [comment]
@Madko I do not think that this is a good idea. If I am using ceph on my virt nodes and install centos-release-openstack-pike the existing ceph packages are overwritten by the ceph packages included in centos-release-openstack-pike. Thus, I prefer to use only centos-release-qemu-ev.

This gives me the best flexibility.

**Comment by Madko** [comment]
Same benefits from using ceph from RDO community you have a large community on a stable fixed ceph version. I have to check but if I remember correctly the ceph package from openstack RDO are in a different repo so it's easy to disable. Or you can exclude ceph from this repo etc. Anyway, that was just a tip for those who want to get a good openvswtich package.
